docs/source/installation/plugins.rst

```diff
@@ -0,0 +1,96 @@
+.. _plugins:
+
+=======
+Plugins
+=======
+
+Plugins are features that extend Manim's core functionality. Since Manim is
+extensible and not everything belongs in its core, we'll go over how to
+install, use, and create your own plugins.
+
+.. note:: 
+
+    The standard naming convention for plugins is to prefix the plugin with
+    ``manim-``. This makes them easy for users to find on package
+    repositories such as PyPI.
+
+.. WARNING::
+
+    The plugin feature is new and under active development. Expect updates
+    for the best practices on installing, using, and creating plugins; as
+    well as new subcommands/flags for ``manim plugins``
+
+Installing Plugins
+******************
+Plugins can be easily installed via the ``pip``
+command:
+
+.. code-block:: bash
+
+    pip install manim-*
+
+After installing a plugin, you may use the ``manim plugins`` command to list
+your available plugins, see the following help output:
+
+.. code-block:: bash
+
+    manim plugins -h
+    usage: manim plugins -h -l
+
+    Utility command for managing plugins
+
+    optional arguments:
+    -h, --help    show this help message and exit
+    -l, --list    Lists all available plugins
+
+    Made with <3 by the manim community devs
```

Comment:
Suggestion: could you please add the output of `manim plugins -l` as well? Mostly for completeness sake, but also because manim tends to attract users that have little or no experience with the command line, so the more examples we include the better.